internal/connector/pingone/mfa/resources/pingone_mfa_application_push_credential_test.go

```diff
@@ -0,0 +1,37 @@
+package resources_test
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/pingidentity/pingctl/internal/connector"
+	"github.com/pingidentity/pingctl/internal/connector/pingone/mfa/resources"
+	"github.com/pingidentity/pingctl/internal/testutils"
+)
+
+func TestMFAApplicationPushCredentialExport(t *testing.T) {
+	// Get initialized apiClient and resource
+	sdkClientInfo := testutils.GetPingOneSDKClientInfo(t)
+	resource := resources.MFAApplicationPushCredential(sdkClientInfo)
+
+	// Defined the expected ImportBlocks for the resource
+	expectedImportBlocks := []connector.ImportBlock{
+		{
+			ResourceType: "pingone_mfa_application_push_credential",
+			ResourceName: "Test MFA_APNS",
+			ResourceID:   fmt.Sprintf("%s/11cfc8c7-ec0c-43ff-b49a-64f5e243f932/7847f8a4-f81e-4994-a095-b4d579deaf52", testutils.GetEnvironmentID()),
```

Comment:
should we have these as env vars instead? Similar to below
https://github.com/pingidentity/terraform-provider-pingfederate/blob/b4ff09210e66141fa0825c3293a3200748cd0028/internal/acctest/config/pingoneconnection/ping_one_connection_test.go#L21

Reply:
Considering the resource ID is only relevant to its environment, I don't think this needs to be an environment variable. Were you concerned about something else?

Reply:
If the environment ID is used throughout multiple files, it could be easily referenced via one environment variable incase the environment changes. We can probably ignore it for now.